core/src/main/java/hudson/triggers/TriggerAdminMonitor.java

```diff
@@ -0,0 +1,79 @@
+package hudson.triggers;
+
+import hudson.ExtensionList;
+import hudson.model.AdministrativeMonitor;
+import hudson.util.CopyOnWriteMap;
+import jenkins.model.Jenkins;
+import org.kohsuke.stapler.HttpResponse;
+import org.kohsuke.stapler.HttpResponses;
+
+import javax.annotation.Nonnull;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Logger;
+
+public class TriggerAdminMonitor extends AdministrativeMonitor {
+
+    private final @Nonnull
+    Map<String, Value> errors = new CopyOnWriteMap.Hash<>();
+
+    private static final Logger LOGGER = Logger.getLogger(TriggerAdminMonitor.class.getName());
+
+    public static TriggerAdminMonitor getInstance() {
+        return ExtensionList.lookup(TriggerAdminMonitor.class).get(0);
+    }
+
+    public TriggerAdminMonitor() {
+    }
+
+    @Override
+    public boolean isActivated() {
+        return !errors.isEmpty();
+    }
+
+    @Override
+    public @Nonnull String getDisplayName() {
+        return "Cron triggers monitor";
+    }
+
+    public void clear() {
+        errors.clear();
+    }
+
+    public void report(@Nonnull String trigger, @Nonnull String msg) {
+        errors.put(trigger, new Value(msg));
+    }
+
+    public @Nonnull Map<String, Value> getErrors() {
+        return new HashMap<>(errors);
+    }
+
+    public HttpResponse doClear() throws IOException {
```

Comment:
Should require post method.

Reply:
Why? It doesn't come from a FORM, but link...

Reply:
That is how we intend that to be used. Though from CSRF protection perspective it can be abused just the same. The rule of thumb is the GET method requests should not modify server state.